images/installer/Dockerfile.rhel7

```diff
@@ -18,26 +18,42 @@ LABEL name="openshift3/ose-ansible" \
       version="v3.6.0" \
       release="1" \
       architecture="x86_64"
+      vcs-url="https://github.com/openshift/openshift-ansible" \
+      vcs-type="git" \
 
 # Playbooks, roles and their dependencies are installed from packages.
 # Unlike in Dockerfile, we don't invoke the 'assemble' script here
-# because all content and dependencies (like 'oc') is already
-# installed via yum.
 USER root
 RUN INSTALL_PKGS="atomic-openshift-utils atomic-openshift-clients python-boto skopeo openssl java-1.8.0-openjdk-headless httpd-tools" && \
     yum repolist > /dev/null && \
     yum-config-manager --enable rhel-7-server-ose-3.6-rpms && \
     yum-config-manager --enable rhel-7-server-rh-common-rpms && \
+    yum install -y --setopt=tsflags=nodocs ansible
```

Comment:
Looking at `INSTALL_PKS`, do we still need some of them? (looking at java in particular)